java/src/com/github/bazel_contrib/contrib_rules_jvm/junit5/BUILD.bazel

```diff
@@ -62,6 +70,15 @@ java_library(
     ],
 )
 
+# Do not build with --release 8 to work around https://bugs.openjdk.org/browse/JDK-8214165.
+java_library(
```

Comment:
These classes specifically don't refer to `SecurityManager` (in fact, they only use `java.lang.reflect.Field`, `java.util.Map`, and `sun.misc.Unsafe`). Is there another reason to specify a fixed `--release`? 

Specifying `--release 11` would break JDK 8 compatibility and specifying `--release 8` isn't possible due to the linked JDK bug.

Reply:
I expanded on why we can't in the comment: `--release 8` breaks due to a JDK bug, `--release N` with N > 8 obviously means we aren't compatible with Java 8 anymore.

The source files in this target reference only very few classes, all of which are extremely unlikely to go away before Java 8 itself, so I don't think it's a problem to not specify `--release` on this target.